src/hacbs/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -15,107 +30,216 @@ type PipelineRunDetailsTabProps = {
+                <DescriptionListDescription>
+                  {getMetadataList(pipelineRun?.metadata?.annotations)}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Created at</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <Timestamp timestamp={pipelineRun?.metadata.creationTimestamp} />
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Duration</DescriptionListTerm>
+                <DescriptionListDescription>{duration}</DescriptionListDescription>
+              </DescriptionListGroup>
+            </DescriptionList>
+          </FlexItem>
+        </Flex>
+
+        <Flex flex={{ default: 'flex_3' }}>
+          <FlexItem>
+            <DescriptionList
+              data-test="pipelinerun-details"
+              columnModifier={{
+                default: '1Col',
+              }}
+            >
+              <DescriptionListGroup>
+                <DescriptionListTerm>Status</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {getBuildStatusIcon(status)} {status}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Message</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {pipelineRun?.status?.conditions[0].message}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Log snippet</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <CodeBlock>
+                    <CodeBlockCode id="code-content">
+                      {
+                        pipelineRun?.status?.taskRuns[`${pipelineRun?.metadata.name}-show-summary`]
+                          .status.taskSpec.steps[0].script
+                      }
+                    </CodeBlockCode>
+                  </CodeBlock>
+                  <Button
+                    variant="link"
+                    isInline
+                    component={(props) => (
+                      <Link
+                        {...props}
+                        to={`/app-studio/pipelineruns/${pipelineRun?.metadata.name}?activeTab=logs`}
+                      />
+                    )}
+                  >
+                    See logs
+                  </Button>
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Pipeline</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {pipelineRun?.metadata.labels[PipelineRunLabel.PIPELINE_NAME]}
```

Comment:
use `-` as a fallback value for any field that may be blank.

Reply:
applied this everywhere applicable, thanks